### PR TITLE
Fix issue in fsfuzzer that it will run in ppc64le

### PR DESCRIPTION
**Pull request by PraveenPenguin**
Signed-off-by: praveen@linux.vnet.ibm.com
Fix this issue 
1-fix config.guess that it able to configure in ppc64le machine
2- fix fstest.c that <attr/xattr.h> include not working and glibc-headers provide /usr/include/sys/xattr.h
3- fix fsfuzz shell script that it work on bash environment

### CODE REVIEW

File: config.guess [hunk 1]
```diff
@@ -937,6 +937,10 @@ EOF
     ppc64:Linux:*:*)
 	echo powerpc64-unknown-linux-gnu
 	exit ;;
+    ppc64le:Linux:*:*)
```
Comment (ldoktor) [inline config.guess:940]: nice

File: fsfuzz [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (ldoktor) [inline fsfuzz:1]: is this change necessary? Simple sh should handle this file correctly, can't it?

Comment (PraveenPenguin) [inline fsfuzz:1]: @ldoktor no with sh environment it is failing  so use bash

Comment (ldoktor) [inline fsfuzz:1]: On RHEL7 it works fine with either of them, so I can't tell, anyway I don't see any harm in this.

**Comment by ldoktor** [comment]
Hello @stevegrubb, would you please take a look at this? Without it it fails on `ppc64le`, with it it seems to work fine.

**Comment by stevegrubb** [comment]
The files that are being patched are pulled in by autogen.sh. I think the best fix is to get rid of them so that they get automatically updated during build. Let me know if you see any other issues.

File: fsfuzz (hunk 2 above)
Comment (PraveenPenguin) [inline fsfuzz:1]: @stevegrubb bash environment works in ubuntu 16.04 can you please adopt this change .
